cluster-operator/pom.xml

```diff
@@ -42,12 +42,16 @@
             <artifactId>vertx-core</artifactId>
         </dependency>
         <dependency>
-            <groupId>org.slf4j</groupId>
-            <artifactId>slf4j-api</artifactId>
+            <groupId>org.apache.logging.log4j</groupId>
+            <artifactId>log4j-api</artifactId>
         </dependency>
         <dependency>
-            <groupId>org.slf4j</groupId>
-            <artifactId>slf4j-log4j12</artifactId>
+            <groupId>org.apache.logging.log4j</groupId>
+            <artifactId>log4j-core</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.logging.log4j</groupId>
+            <artifactId>log4j-slf4j-impl</artifactId>
```

Comment:
This is necessary because `io.fabric8:kubernetes-client` depends on `org.slf4j:jul-to-slf4j`? Because AFAICS that's the only other thing depending on slf4j in the CO. If so, I wonder if we could remove the `org.slf4j:jul-to-slf4j` dependency and replace it with https://logging.apache.org/log4j/2.0/log4j-jul/index.html, so avoid having to go via slf4j?